backend/template_mail.json

```diff
@@ -0,0 +1,14 @@
+{
+    "emails": {
+        "grant_access": {
```

Comment:
Please rename the templates to access_grant, access_revoke etc so the context comes first and then the action

Reply:
Done